client/src/components/UserSettings/SettingsTagColor.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
```

Comment:
I think `TagColor` file name will be a better option. It will avoid duplications and shorten import paths at least.

Reply:
and it's done too